packages/camera/camera_android/android/src/test/java/io/flutter/plugins/camera/features/exposurepoint/ExposurePointFeatureTest.java

```diff
@@ -310,4 +312,21 @@ public void updateBuilder_shouldNotSetMeteringRectangleWhenNoValidCoordsAreSuppl
     exposurePointFeature.updateBuilder(mockCaptureRequestBuilder);
     verify(mockCaptureRequestBuilder, times(3)).set(any(), isNull());
   }
+
+  @Test
+  public void testShouldResetReturnsFalse() {
```

Comment:
Can you also add a test for when it should return true?

Reply:
Yes, i did